lib/kibana/kibana/connector.py

```diff
@@ -89,7 +89,7 @@ def url(self, uri):
         # If a space is defined update the URL accordingly
         uri = uri.lstrip('/')
         if self.space:
-            uri = "s/{}/{}".format(self.space, uri)
+            uri = "s/{}/{}".format(self.space.lower(), uri)
```

Comment:
better to set it in `__init__` than on this call

```diff
-        self.space = space if space and space.lower() != 'default' else None 
+        self.space = space.lower() if space and space.lower() != 'default' else None
```

Reply:
The problem is that we do a check later on again the space name so we probably shouldn't do this.